src/words/fast.jl

```diff
@@ -95,6 +95,22 @@ a word boundary should follow `s`.
     false
 """
 function lookahead(ts::TokenBuffer, s; boundary = false)
+  if typeof(s) == Regex
```

Comment:
This can  AFAICT be a seperate method.

`lookahead(ts::TokenBuffer, s::Regex; boundary = false)`

Reply:
This will be good to break using dispatch